trap/src/main/java/com/cursorinsight/trap/datasource/gesture/internal/TrapWindowCallback.kt

```diff
@@ -11,6 +11,7 @@ import android.view.View
 import android.view.Window
 import android.view.WindowManager
 import android.view.accessibility.AccessibilityEvent
+import com.cursorinsight.trap.TrapConfig
```

Comment:
Is this needed here?

Reply:
No, I've removed all the unused imports from this file.